src/systemcmds/tests/test_parameters.cpp

```diff
@@ -157,14 +178,186 @@ bool ParameterTest::ResetAllExcludesWildcard()
 	return ret;
 }
 
+bool ParameterTest::exportImport()
+{
+	bool ret = true;
+
+	param_t test_params[] = {p0, p1, p2, p3};
+
+	// set all params to corresponding param_t value
+	for (auto p : test_params) {
+		if (param_type(p) == PARAM_TYPE_INT32) {
+			int set_val = p;
+			param_set_no_notification(p, &set_val);
+
+			ret = ret && _assert_parameter_int_value(p, p);
+		}
+
+		if (param_type(p) == PARAM_TYPE_FLOAT) {
+			float set_val = p;
```

Comment:
maybe something like `(float)p + 0.489f;` to be sure it's a float

Reply:
Makes sense, done.